src/web/event-notifications/event-notification-types/SlackNotificationForm.jsx

```diff
@@ -141,9 +174,31 @@ class SlackNotificationForm extends React.Component {
                  label="Custom Message (optional)"
                  type="textarea"
                  bsStyle={validation.errors.custom_message ? 'error' : null}
-                 help={get(validation, 'errors.custom_message[0]', 'Custom message to be appended below the alert title. See https://docs.graylog.org/en/latest/pages/alerts.html#data-available-to-notifications for more details.')}
+                 help={get(validation, 'errors.custom_message[0]', element)}
                  value={config.custom_message || ''}
                  onChange={this.handleChange} />
+
+          <FormGroup>
+            <ControlLabel>Backlog Message Items (optional)</ControlLabel>
+            <InputGroup>
+              <InputGroup.Addon>
+                <input id="toggle_backlog_size"
+                       type="checkbox"
+                       checked={isBacklogSizeEnabled}
+                       onChange={this.toggleBacklogSize} />
+              </InputGroup.Addon>
+              <FormControl type="number"
+                           id="backlog_size"
+                           name="backlog_size"
+                           onChange={this.handleBacklogSizeChange}
+                           value={backlogSize}
+                           min="0"
+                           max="50"
+                           disabled={!isBacklogSizeEnabled} />
+            </InputGroup>
+            <HelpBlock>Slack notification cannot have more than 50 backlog messages.</HelpBlock>
```

Comment:
This does not do a very good job of explaining what this field is for.  Also, I don't think we should be putting an upper limit on the value.  This `HelpBlock` should describe what this field is for and also make it clear what will happen when set to a special value (zero).

Reply:
the docs has more verbose explanation.

Reply:
please refer screen shot above, just so we are consistent, with the parent field name and description.